CONTRIBUTING.md

```diff
@@ -9,17 +9,21 @@
 0. Start postgres if it isn't automatically running: `postgres -D /usr/local/var/postgres`
```

Comment:
let's remove `Copy the secret.rb.template to secret.rb. See below CROMERR Signing` while we're here